src/rules/utils.ts

```diff
@@ -652,6 +652,16 @@ export const isDescribe = (
   );
 };
 
+export const getNodeTitle = (node: TSESTree.CallExpression): string | null => {
+  const [argument] = node.arguments;
+
+  if (!isStringNode(argument) || AST_NODE_TYPES.Literal !== argument.type) {
```

Comment:
iirc I think this can be replaced w/ `isValidAccessor` (might not be the exact name) - `getAccessorValue` definitely has a complementary guard that should cover all the expected conditions for string parameters.